### PR TITLE
updated docs for 1.3

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

**Comment by stof** [comment]
:+1:

File: doc/cookbook/sub_requests.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ By calling ``handle``, you can make a sub-request manually. Here's an example::
 
 There's some more things that you need to keep in mind though. In most cases
 you will want to forward some parts of the current master request to the
-sub-request. That includes: Cookies, server information, session.
+sub-request like cookies, server information, or the session.
```
Comment (SpacePossum) [inline doc/cookbook/sub_requests.rst]: `, or the sesion.` Serial comma not matching http://symfony.com/doc/current/contributing/documentation/standards.html ;)

**Comment by SpacePossum** [comment]
:+1:

File: doc/services.rst [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Services
 ========
 
-Silex is not only a microframework. It is also a micro service container. It
-does this by extending `Pimple <http://pimple.sensiolabs.org>`_ which provides
-service goodness in just 44 NCLOC.
+Silex is not only a framework, it is also a service container. It does this by
+extending `Pimple <http://pimple.sensiolabs.org>`_ which provides service
+goodness in just 44 NCLOC.
```
Comment (GromNaN) [inline doc/services.rst]: That's more 120 NCLOC nowadays.

File: doc/usage.rst [hunk 3]
```diff
@@ -65,17 +63,14 @@ Routing
...
+route is matched. A route pattern consists of:
 
 * *Pattern*: The route pattern defines a path that points to a resource. The
   pattern can include variable parts and you are able to set RegExp
   requirements for them.
 
 * *Method*: One of the following HTTP methods: ``GET``, ``POST``, ``PUT`` or
-  ``DELETE``. This describes the interaction with the resource. Commonly only
-  ``GET`` and ``POST`` are used, but it is possible to use the others as well.
+  ``DELETE``. This describes the interaction with the resource.
```
Comment (GromNaN) [inline doc/usage.rst]: You can add the `PATCH` method.

File: doc/services.rst (hunk 2 above)
Comment (GromNaN) [inline doc/services.rst]: Response to myself, that's Silex 2 That will use Pimple.

**Pull request merged**